addons/mail/static/src/messaging/test_utils.js

```diff
@@ -601,40 +634,21 @@ async function start(param0 = {}) {
         destroy: destroyCallbacks,
         return: returnCallbacks,
     } = callbacks;
-    const { debug = false } = param0;
     const {
-        messagingEnvExtension,
-        services = getServices({ hasChatWindow, debug }),
-        session = {},
-        'window.innerHeight': windowInnerHeight,
-        'window.innerWidth': windowInnerWidth,
-        'window.Notification': windowNotification,
+        debug = false,
+        env,
     } = param0;
+    const {
+        services = getMailServices({ env, hasChatWindow, debug }),
+    } = param0;
+    const session = services.messaging.env;
```

Comment:
I think this should be it but somehow it doesn't break anything
```suggestion
    const { session } = services.messaging.env;
```

Reply:
Yes, I think nothing breaks because code relies on session in env, not how session is passed to mocked environment. That means `web.session` is mis-configured during test, but not `this.env.session`.

Reply:
I tried to pass the value correctly but it made bugs. Since it isn't required and it works fine without it I just removed it.